lib/commands/list-meetings.js

```diff
@@ -0,0 +1,37 @@
+const BBB = require("../BBB");
+const { getLogger } = require("../command-helpers");
+const _ = require("lodash/fp");
+
+module.exports = {
+  name: "list-meetings",
+
+  description: "List meetings currently running on BBB",
+
+  options: (yargs) => yargs,
+
+  handler: (argv) => {
+    const logger = getLogger(argv.verbose);
+    const bbbClient = new BBB(argv.u, argv.s);
+
+    bbbClient
+      .getMeetings()
+      .then((meetings) => {
+        if (meetings.length === 0) {
+          logger.info("No meeting running on the BBB server");
+          return;
+        }
+        logger.info("Currently running meetings :");
+        for (const meeting of meetings) {
+          logger.info(
+            ` - ${meeting.meetingID} (${meeting.meetingName}) – participants : ${meeting.participantCount}`
+          );
+          const attendees = [].concat(_.get("attendees.attendee")(meeting));
+          if (attendees.length > 0) {
+            const attendeesNames = _.map("fullName")(attendees);
+            logger.debug(`   [${attendeesNames.join(", ")}]`);
```

Comment:
```suggestion
            logger.debug(`[${attendeesNames.join(", ")}]`);
```

Reply:
I keep it like this to have a consistent list output : 

```
info:     - meeting-id1 (Meeting 1) – participants : 1
debug:      [Matthieu]
info:     - meeting-id2 (Home Room) – participants : 1
debug:      [Matthieu]
info:     - meeting-id3 (Meeting 3) – participants : 0
debug:      []
```

With your suggestion I would get : 
```
info:     - meeting-id1 (Meeting 1) – participants : 1
debug:   [Matthieu]
info:     - meeting-id2 (Home Room) – participants : 1
debug:   [Matthieu]
info:     - meeting-id3 (Meeting 3) – participants : 0
debug:   []
```